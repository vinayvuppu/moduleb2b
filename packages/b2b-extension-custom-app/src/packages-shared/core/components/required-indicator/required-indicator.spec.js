import React from 'react';
import { shallow } from 'enzyme';
import RequiredIndicator from './required-indicator';

describe('with default props', () => {
  it('should match snapshot', () => {
    expect(shallow(<RequiredIndicator />)).toMatchSnapshot();
  });
});

describe('when uncolored', () => {
  it('should match snapshot', () => {
    expect(shallow(<RequiredIndicator uncolored={true} />)).toMatchSnapshot();
  });
});
