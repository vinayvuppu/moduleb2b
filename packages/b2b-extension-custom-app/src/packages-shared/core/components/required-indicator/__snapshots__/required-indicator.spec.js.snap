// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when uncolored should match snapshot 1`] = `
<em
  className=""
>
  *
</em>
`;

exports[`with default props should match snapshot 1`] = `
<em
  className="colored"
>
  *
</em>
`;
